.github/scripts/setup_runners.sh

```diff
@@ -119,10 +121,15 @@ then
                          --parameters '{"commands": ["#!/bin/bash", "sudo hostnamectl set-hostname '${instance_name2}'"]}' \
                          --output text > /dev/null 2>&1; echo $?
 
+    echo "[${instance_name2}]: Get latest runner binary to server ${RUNNER_URL}"
+    aws ssm send-command --targets Key=tag:Name,Values=$instance_name2 --document-name "AWS-RunShellScript" \
+                         --parameters '{"commands": ["#!/bin/bash", "sudo su - '${SETUP_AMI_USER}' -c \"mkdir -p '${RUNNER_DIR}' && cd '${RUNNER_DIR}' && wget -q '${RUNNER_URL}' && tar -xzf *.tar.gz && rm *.tar.gz \""]}' \
+                         --output text > /dev/null 2>&1; echo $?
+
     echo "[${instance_name2}]: Get runner token and bootstrap on Git"
     instance_runner_token=`curl --silent -H "Authorization: token ${SETUP_TOKEN}" --request POST "${GIT_URL_API}/${GIT_URL_REPO}/actions/runners/registration-token" | jq -r .token`
     aws ssm send-command --targets Key=tag:Name,Values=$instance_name2 --document-name "AWS-RunShellScript" \
-                         --parameters '{"commands": ["#!/bin/bash", "sudo su - '${SETUP_AMI_USER}' -c \"cd actions-runner && ./config.sh --unattended --url '${GIT_URL_BASE}/${GIT_URL_REPO}' --labels '${instance_name2}' --token '${instance_runner_token}' && nohup ./run.sh &\""]}' \
+                         --parameters '{"commands": ["#!/bin/bash", "sudo su - '${SETUP_AMI_USER}' -c \"cd '${RUNNER_DIR}' && sleep 10 && ./config.sh --unattended --url '${GIT_URL_BASE}/${GIT_URL_REPO}' --labels '${instance_name2}' --token '${instance_runner_token}' && nohup ./run.sh &\""]}' \
```

Comment:
what is the reason to run ./run.sh and not ./svc.sh? Won't it be a better option to run it as a service rather than running a script in the background?

Reply:
If this is a long term server, service is the solution. However, this server/runner will be decommissioned immediately after complete the integTest, so we can simply run it.

Reply:
I hope that it this process is not affected by any other process. For example, elasticsearch.service (high cpu utilization), etc